private/rdf4cpp/regex/RegexImpl.cpp

```diff
@@ -20,9 +20,42 @@ static re2::RE2::Options translate_flags(Regex::flag_type const flags) {
     return o;
 }
 
+static re2::RE2 build_regex(std::string_view regex, Regex::flag_type flags) {
+    auto opt = translate_flags(flags);
+    if (!flags.contains(RegexFlag::Multiline) && !flags.contains(RegexFlag::RemoveWhitespace)) {
+        return {regex, opt};
+    }
+    std::string x{};
+    x.reserve(regex.size()+4);
+    if (flags.contains(RegexFlag::Multiline)) {
+        x.append("(?m)");
```

Comment:
Is this one different from `Options::set_one_line`?

Reply:
`Options::set_one_line` does not work, if we use re2 in its current syntax mode.

Reply:
thats unfortunate, but is fine